boxsdk/object/base_object.py

```diff
@@ -1,34 +1,18 @@
 # coding: utf-8
 
-from __future__ import unicode_literals
-from abc import ABCMeta
+from __future__ import unicode_literals, absolute_import
 import json
 
-import six
+from .base_endpoint import BaseEndpoint
+from.base_api_json_object import BaseAPIJSONObject
+from ..util.translator import Translator
 
-from boxsdk.object.base_endpoint import BaseEndpoint
-from boxsdk.util.translator import Translator
 
+class BaseObject(BaseEndpoint, BaseAPIJSONObject):
+    """A Box API endpoint for interacting with a Box object."""
 
-class ObjectMeta(ABCMeta):
-    """
-    Metaclass for Box API objects. Registers classes so that API responses can be translated to the correct type.
-    Relies on the _item_type field defined on the classes to match the type property of the response json.
-    But the type-class mapping will only be registered if the module of the class is imported.
-    So it's also important to add the module name to __all__ in object/__init__.py.
-    """
-    def __init__(cls, name, bases, attrs):
-        super(ObjectMeta, cls).__init__(name, bases, attrs)
-        item_type = attrs.get('_item_type', None)
-        if item_type is not None:
-            Translator().register(item_type, cls)
-
-
-@six.add_metaclass(ObjectMeta)
-class BaseObject(BaseEndpoint):
-    """
-    A Box API endpoint for interacting with a Box object.
-    """
+    # Question:
+    # Should this item type be a part of other objects?
```

Comment:
No, it isn't necessary. This is just here so that all of its subclasses at least have `_item_type` defined, even if it is `None`.

Though this should move to `BaseAPIJSONObject`, since that is now the base for translation.